checker/tests/tainting/HasQualParamDefaults.java

```diff
@@ -44,6 +44,21 @@ public Buffer append(@PolyTainted String s) {
             someString = s;
             return s;
         }
+
+        void initializeLocalTainted(@Tainted Buffer b) {
+            Buffer local = b;
+            @Tainted Buffer copy = local;
+        }
+
+        void initializeLocalUntainted(@Untainted Buffer b) {
+            Buffer local = b;
+            @Untainted Buffer copy = local;
```

Comment:
How about adding a test with a local variable without initializer, showing how that will produce an expected error when the default and the RHS-type mismatch?

Reply:
More generally, maybe a test with an expected error would be good, to show that the expected type is inferred.